relayer/src/connection.rs

```diff
@@ -754,9 +755,11 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Connection<ChainA, ChainB> {
     pub fn build_conn_init_and_send(&self) -> Result<IbcEvent, ConnectionError> {
         let dst_msgs = self.build_conn_init()?;
 
+        let tm = TrackedMsgs::new(dst_msgs, "create connection");
```

Comment:
MZ: COuld be an enum, not a hardcoded string.

AS: These changes are not very interesting, just a workaround to using tracked messages. This code pertains to interactive CLIs (e.g., `hermes create channel --port-a transfer..`).